spec/migrations/20190617191109_move_awx_credentials_to_authentications_spec.rb

```diff
@@ -251,34 +259,77 @@
 
         expect_authentications_migrated
       end
+
+      context "with a 'bogus' secret key (support use case)" do
```

Comment:
Remove the "support use case" text because it doesn't make sense in upstream.

Reply:
switched it to "(DB restore use case)" since I think it is still worth mentioning a distinction the spec description.